bookkeeper-dist/server/pom.xml

```diff
@@ -31,7 +31,80 @@
   <packaging>jar</packaging>
   <name>Apache BookKeeper :: Dist (Server)</name>
 
+  <properties>
+    <disruptor.version>3.4.0</disruptor.version>
```

Comment:
this should be here

also, why do you want to add this dependency ?

Reply:
@eolivelli 
log4j-core need disruptor to write log, so import disruptor like pulsar
<img width="686" alt="image" src="https://user-images.githubusercontent.com/42990025/188871331-880a8d93-01f2-4fc0-99af-856fa55dee89.png">